packages/echo-core/src/settings/moduleSettings.ts

```diff
@@ -70,7 +70,7 @@ export class ModuleSettings<T> {
      * @param {Partial<T>} initialState
      * @memberof ModuleSettings
      */
-    protected setup<T>(key: string, initialState: Partial<T>): void {
+    protected setup(key: string, initialState: Partial<T>): void {
```

Comment:
this can be breaking if someone has inherited this, and used setup<foo>? But it's not very likely, so should be ok

Reply:
Hmmm, did a double check on this.

As I remembered here was a typescript error, that's why I needed to remove it. But now I re-added it, and it's all ok.

So I'll keep this type template here, just to be on the safe side. Thanks for pointing this out!